README.md

```diff
@@ -58,224 +61,39 @@ python setup.py install
-    * Visualization of the physical systems state, reference and rewards.
-* State Filter
-    * keyword: `state_filter`
-    * Selection of states that shall be shown to the agent.
-    
-These function modules can be selected in three ways:
-
-* Passing a keystring (and further keyword arguments for the class) :
-     * `reference_generator='SinusoidalReference', amplitude_range=(0.2, 0.8)`
-
-* Passing a class pointer (and further keyword arguments for the class)
-    * `reference_generator=SinusoidalReferenceGenerator, amplitude_range=(0.2,0.8)`
-
-* Passing an instantiated object
-  * `reference_generator = SinusoidalReferenceGenerator(amplitude_range=(0.2,0.8)`
-      
-Furthermore, the internal function modules of the physical systems like the converter, motor, load can be selected in
-the make keyword-arguments in the same way. 
-
-The available modules and specific keyword-arguments for each module can be looked up in the API-documentation.
-### Reset
-The reset function determines new references, new initial values and resets the visualization.
-Call this function before a new episode begins.
-The parameters of the motor, converter and load will be those during instantiation.
-
-### Step
-This function performs one action on the environment for one time step.
-It simulates the motor and needs to be called in every time step. It takes the action as parameter only.
-First the input voltage to the motor from the converter is determined and afterwards an integrator is used to compute 
-the next state. 
-Eventually, the reward is evaluated and returned together with the next observation and a flag indicating termination.
-Several reward functions are available.
-
-### Render
-The visualization contains graphs of the motor quantities 'speed, voltages, currents, torque' for one episode. 
-What should be shown is to be specified in the configuration-parameter.
-The quantities that should be displayed can be specified in the constructor-parameters.
-All visualizations are optional and recommended to be disabled for increased speed of training.
-
-### Examples
-
-- Conventional PI controller as speed controller on a dc series motor [(jump to source)](examples/pi_series_omega_control.py).
-
-- Training and testing of a [Keras-rl](https://github.com/keras-rl/keras-rl) DDPG-Agent as a speed controller on a dc series motor [(jump to source)](examples/ddpg_series_omega_control.py).
- 
-### Physical System Models
-The following electrical motor , converter and mechanical load models are included. More detailed descriptions can be found in the corresponding classes.
- 
-##### Motor Models
-The following motor models are included:
+Among various DC-motor models, the following three-phase motors - together with their power electronic counterparts - are available:
+- Permanent Magnet Synchronous Motor (PMSM), 
+- Synchronous Reluctance Motor (SynRM)
+- Squirrel Cage Induction Motor (SCIM)
+- Doubly-fed Induction Motor (DFIM)
 
```

Comment:
a motor type is not a proper noun, therefore this list should be in small letters